mockDB.js

```diff
@@ -0,0 +1,103 @@
+var config = require('./config.js'),
+    crudService = require('./scripts/crud-service.js'),
+    doc = require('./scripts/doc-config.js'),
+    usage = require('./scripts/usage.js');
+
+var url = config.db.url,
+    cmds = [
+        'view',
+        // 'find',
+        'insert',
+        'delete',
+        // 'replace',
+        // 'update'
+    ],
+    collectionNames = [
+        'users',
+        'posts'
+    ],
+    args = process.argv.slice(2);
+
+/*  Service calls to 'user-service'.  Note that
+    parameters userData, query, replacement, and updatedData
+    should be in JSON format */
+var db = {
+
+    find: function(collection, query) {
```

Comment:
These methods belong in your CMS no?  This app should only focus on generating / deleting content from user DB.  I  think we need to recap what the goals of each of the 3 projects should be.